flipperdevices/flipperzero-firmware
# [FL-1976] BLE HID

# What's new

- Add BLE keyboard to debug application
- Add switch ble profile API

# Verification 

- Open ble keyboard app, connect to computer. Verify flipper now in BLE Keyboard mode.
- Close app. Verify flipper now in Ble Serial mode

# Checklist (do not modify)
